todoapp/todos/models.py

```
@@ -1,18 +1,33 @@
 
-        Add string representation for this model with todos name.
+    Add string representation for this model with todos name.
     """
+
+    name = models.CharField(max_length=1000, blank=False)
+    user = models.ForeignKey(settings.AUTH_USER_MODEL, on_delete=models.CASCADE)
+    done = models.BooleanField(default=False)
+    date_created = models.DateTimeField(auto_now_add=True, editable=False)
+    date_completed = models.DateTimeField(null=True, blank=True, editable=False)
+
+    def __str__(self):
```

Ruff (0.12.2)

Missing return type annotation for special method `__str__`

Add return type annotation: `str`

(ANN204)